### PR TITLE
Instruction to install node via nvm

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary

Added instruction for installing node via nvm

#### Ticket Link

As per recommendation from https://github.com/mattermost/mattermost-webapp/pull/9335#issuecomment-966427513

### CODE REVIEW

File: site/content/contribute/webapp/developer-setup.md [hunk 1]
```diff
@@ -11,7 +11,10 @@ Set up your development environment for building, running, and testing the Matte
 
 2. Install Node.js 16:
 
-    - If you use [NVM](https://github.com/nvm-sh/nvm), use that to install Node 16.
+    - If you use [NVM](https://github.com/nvm-sh/nvm), use that to install Node 16. Following command can be run inside `matterermost-webapp` folder once you have cloned it, this will install the correct Node version i.e 16.
```
Comment (cwarnermm) [inline site/content/contribute/webapp/developer-setup.md:14]:
```suggestion
    - If you use [NVM](https://github.com/nvm-sh/nvm), use that to install Node 16. Once you've cloned the `matterermost-webapp`repo, run following command inside the `matterermost-webapp` folder to install the correct Node version i.e 16.
```

**Review by cwarnermm — COMMENTED** [review]
Thank you for this documentation update! I've provided feedback inline. Please let me know if you have any questions or concerns.

File: site/content/contribute/webapp/developer-setup.md (hunk 1 above)
Comment (M-ZubairAhmed) [reply site/content/contribute/webapp/developer-setup.md]: Thank you, I had added your suggestion

**Review by cwarnermm — APPROVED** [review]
Thanks!

**Comment by hmhealey** [comment]
@cwarnermm I re-requested review since I reordered the steps a bit. I wanted to avoid the user having to remember to run `nvm install` after doing some of the other setup steps

**Review by cwarnermm — APPROVED** [review]
Reads well! Thanks!

**Pull request merged**